src/main/java/racingcar/IOClass.java

```diff
@@ -0,0 +1,77 @@
+package racingcar;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class IOClass {
+
+    public static Car[] inputName() {
+        System.out.println("경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분).");
+        Scanner sc = new Scanner(System.in);
+        String namesString = sc.nextLine();
+        namesString = namesString.replace(" ", "");
+        String[] carNames = namesString.split(",");
+        final int CARS_COUNT = carNames.length;
+        Car[] cars = new Car[CARS_COUNT];
+        for (int i = 0; i < CARS_COUNT; i++) {
+            cars[i] = new Car(carNames[i]);
+        }
+
+        return cars;
+    }
+
+    public static boolean checkCarName(Car[] cars) {
+        final int CARS_COUNT = cars.length;
+        for (int i = 0; i < CARS_COUNT; i++) {
+            if (cars[i].getName().length() > 5) {
+                return false;
+            }
+        }
+        return true;
+    }
+
+    public static int getTimes() {
+        System.out.println("시도할 횟수는 몇 회인가요?");
+        Scanner sc = new Scanner(System.in);
+        int times = sc.nextInt();
+        return times;
+    }
+
+    public static void printCurrentPosition(Car[] cars) {
+        final int CARS_COUNT = cars.length;
+        for (int i = 0; i < CARS_COUNT; i++) {
+            System.out.print(cars[i].getName() + " : ");
+            for (int j = 0; j < cars[i].getPosition(); j++) {
+                System.out.print("-");
+            }
+            System.out.println();
+        }
```

Comment:
`향상된 for문`으로 변경해보면 어떨까요? 🤗 
```suggestion
        for (Car car : cars) {
            System.out.print(car.getName() + " : ");
            for (int j = 0; j < car.getPosition(); j++) {
                System.out.print("-");
            }
            System.out.println();
        }
```